iotlabcli/tests/c23.py

```diff
@@ -34,11 +34,13 @@
     from urllib2 import HTTPError
     import mock
     from cStringIO import StringIO
+    reload_module = reload  # pylint:disable=invalid-name, undefined-variable
```

Comment:
This make the test super complicated but I understand you made it for coverage. A simpler solution is to add another tox environment that will install iotlabsshcli, serial_aggregator, etc.
Normally codecov is able to merge coverage results from multiple tox environments.

Reply:
The small issue is that serial_aggregator or oml plot tools are not on pypi, so installing would be a git clone away, not very less complicated IMO. Could be solved by deploying them on pypi

Reply:
> Could be solved by deploying them on pypi

No there's no need to deploy on pipy, you can use an http url, using a requirements.txt file or maybe even in the tox environment.

Reply:
OK, i'll look into it, to get coverage without the reload_module ;-)